mesonbuild/modules/python.py

```diff
@@ -442,8 +442,7 @@ def install_sources_method(self, args: T.Tuple[T.List[T.Union[str, mesonlib.File
         return self.interpreter.install_data_impl(
             self.interpreter.source_strings_to_files(args[0]),
             self._get_install_dir_impl(kwargs['pure'], kwargs['subdir']),
-            mesonlib.FileMode(),
-            None)
+            mesonlib.FileMode(), rename=None, tag='runtime')
```

Comment:
Noticed this in the latest push... shouldn't this allow specifying a user-set tag? Maybe the tag should be python-related, this would also be convenient for marking installable files for consumption by mesonpep517, cross-reference #7863

Reply:
I should add `install_tag` kwarg there too, indeed. Maybe I should change the default to `python-runtime` to make them installable separately? I'm not sure, because my understanding is you could install a python executalbe to `/usr/bin` (that would be tagged as `runtime`) and it requires its modules to be installed by `install_sources()` ?

Reply:
Added `install_tag` kwarg to `python.install_sources()`.

Reply:
In that case, one would need to manually tag the python executable as `install_tag: 'python-runtime'`

Reply:
Changed the default to `python-runtime` after discussion on IRC.